src/client/Client.h

```diff
@@ -1131,11 +1131,6 @@ class Client : public Dispatcher, public md_config_obs_t {
   int get_caps_issued(int fd);
   int get_caps_issued(const char *path, const UserPerm& perms);
 
-  // low-level interface v2
-  inodeno_t ll_get_inodeno(Inode *in) {
-    Mutex::Locker lock(client_lock);
-    return _get_inodeno(in);
```

Comment:
Might not hurt to get rid of the _get_inodeno wrapper too, while you're in there. Seems rather silly to have a private method for fetching a single value out of the class.